### PR TITLE
Add announcement for dropping PyPy 3.7

### PR DESCRIPTION
**Pull request by h-vetinari**
PR Checklist:

- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

As discussed in the dev meeting, announcing the drop of pypy 3.7, see #1800 & https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3227

Note that to avoid rebasing issue, this currently includes #1810.

CC @conda-forge/help-pypy

### CODE REVIEW

**Comment by h-vetinari** [comment]
Seems Isuru might have found a way around this. Might want to wait to see how https://github.com/conda/menuinst/pull/103 turns out, at least regarding the part about dropping pypy support in miniforge.

**Comment by isuruf** [comment]
Can you rebase this and remove the part about dropping miniforge pypy support?

File: src/user/announcements.rst [hunk 1]
```diff
@@ -8,6 +8,14 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
...
 ----
 
+2022-08-11: Dropping PyPy 3.7
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-forge has supported PyPy since almost 2.5 years now, and the initial
+PyPy 3.7 builds have been superseded in almost all aspects by the newer builds
+for 3.8 & 3.9. Dropping PyPy 3.7 has the consequence that we are currently
+unable to keep supporting the miniforge installer with PyPy in the base
+environment, but you can keep using PyPy for any other environment!
```
Comment (h-vetinari) [inline src/user/announcements.rst:18]:
```suggestion
```

File: src/user/announcements.rst [hunk 2]
```diff
@@ -8,6 +8,14 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+2022-08-11: Dropping PyPy 3.7
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-forge has supported PyPy since almost 2.5 years now, and the initial
+PyPy 3.7 builds have been superseded in almost all aspects by the newer builds
+for 3.8 & 3.9. Dropping PyPy 3.7 has the consequence that we are currently
+unable to keep supporting the miniforge installer with PyPy in the base
```
Comment (h-vetinari) [inline src/user/announcements.rst:17]:
```suggestion

```

File: src/user/announcements.rst [hunk 3]
```diff
@@ -8,6 +8,14 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+2022-08-11: Dropping PyPy 3.7
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-forge has supported PyPy since almost 2.5 years now, and the initial
+PyPy 3.7 builds have been superseded in almost all aspects by the newer builds
+for 3.8 & 3.9. Dropping PyPy 3.7 has the consequence that we are currently
```
Comment (h-vetinari) [inline src/user/announcements.rst:16]:
```suggestion
for 3.8 & 3.9.
```

**Review by h-vetinari — COMMENTED** [review]
Changes

File: src/user/announcements.rst [hunk 4]
```diff
@@ -8,6 +8,12 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+2022-08-11: Dropping PyPy 3.7
```
Comment (h-vetinari) [inline src/user/announcements.rst:11]:
```suggestion
2022-08-17: Dropping PyPy 3.7
```

**Comment by h-vetinari** [comment]
Can't rebase because I had to do this from a phone, but you should be able to squash merge. :)

File: src/user/announcements.rst [hunk 5]
```diff
@@ -8,6 +8,12 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+2022-08-17: Dropping PyPy 3.7
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-forge has supported PyPy since almost 2.5 years now, and the initial
+PyPy 3.7 builds have been superseded in almost all aspects by the newer builds
+for 3.8 & 3.9.
```
Comment (h-vetinari) [inline src/user/announcements.rst:16]:
```suggestion
for 3.8 & 3.9. We are therefore dropping PyPy 3.7 as a supported python version,
and will keep focussing on the more contemporary PyPy builds.
```

File: src/user/announcements.rst [hunk 6]
```diff
@@ -8,6 +8,13 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+2022-08-17: Dropping PyPy 3.7
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-forge has supported PyPy since almost 2.5 years now, and the initial
+PyPy 3.7 builds have been superseded in almost all aspects by the newer builds
+for 3.8 & 3.9. We are therefore dropping PyPy 3.7 as a supported python version,
+and will keep focussing on the more contemporary PyPy builds.
```
Comment (isuruf) [inline src/user/announcements.rst:17]:
```suggestion
and will keep focusing on the more contemporary PyPy builds.
```

**Pull request merged**